src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -2167,8 +2167,8 @@ subroutine ocn_validate_state(domain, timeLevel)!{{{
                call ocn_write_field_statistics(debugUnit, fieldName, minValue, maxValue)
             end if
 
-            ! Test snowFlow
-            fieldName = 'snowFlow'
+            ! Test snowFlux
+            fieldName = 'snowFlux'
```

Comment:
This one seems like an actual bug, not just a typo.

Reply:
OK. See #175.